src/build_log.cc

```diff
@@ -12,6 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// On AIX, inttypes.h gets indirectly included by build_log.h.
+// It's easiest just to ask for the printf format macros right away.
+#ifndef _WIN32
```

Comment:
I simply moved the `#define` up from lower down the file. It was previously guarded by `#ifndef _WIN32`.